gradle.properties

```diff
@@ -1,3 +1,3 @@
-org.gradle.parallel=true
-org.gradle.configureondemand=true
-org.gradle.cache.tasks=true
+org.gradle.parallel=false
+org.gradle.configureondemand=false
+org.gradle.cache.tasks=false
```

Comment:
Sure, we can do this, but why is this necessary? The gradle 4.0 default is now to run with parallel true.